public/static/locales/en-US/landing.json

```diff
@@ -29,6 +29,27 @@
         "gotIt": "Got it"
       }
     },
+    "patchNotes": {
```

Comment:
In the json file, you added a feature object to contain the feature cards info. Very self-explanatory in my opinion